src/pyhf/parameters/paramview.py

```diff
@@ -93,15 +93,19 @@ def slices(self):
         """
         return self._slices
 
-    def get(self, tensor):
+    def get(self, tensor, indices=None):
```

Comment:
Would be good to docstring the arguments, but again, I can do this after the benchmark testing on this PR is done.

Reply:
done